docs/src/app/new/(content)/components/field/demos/hero/tailwind/index.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Field } from '@base-ui-components/react/field';
+
+export default function ExampleField() {
+  return (
+    <Field.Root className="flex flex-col gap-2">
+      <div className="flex flex-col items-start gap-1">
+        <Field.Label className="text-sm font-medium text-gray-900">Name</Field.Label>
+        <Field.Control
+          required
+          // @ts-ignore
+          placeholder="Required"
+          className="w-64 rounded-md border border-gray-200 px-3.5 py-2 text-gray-900 focus:outline-2 focus:-outline-offset-1 focus:outline-blue-800"
+        />
```

Comment:
Something is off with `Field.Control` type definitions, it errors for `placeholder`

Reply:
This is because it also tries to support types of `select` and `textarea` along with `input` as these can be rendered as a control as well. Maybe it should just use `input` types?